### PR TITLE
ChangelogUpdatesforv1.5

### PR DESCRIPTION
**Pull request by Zuhairahmed**
## Description

Added new in Change Log format inspired by the one in [HasiCorp Terraform repo](https://github.com/hashicorp/terraform/blob/main/CHANGELOG.md); general cleanup 

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the Terraform contribution guidelines
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run make fmt and formatted my code

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,38 +4,41 @@
 
 **Fixed**
 
+ENHANCEMENTS: 
 - INTMDB-224	Support AtlasGov with Terraform [\#865](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/865)
 - INTMDB-314	Feature add: Add ability to upgrade shared/TENANT tiers for clusters and advanced clusters [\#874](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/874)
+- INTMDB-349  New AtlasGov parameter to tell the provider to use the Atlas gov base URL [\#865](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/865)
```
Comment (themantissa) [inline CHANGELOG.md:10]: Non-blocking nit - missing some spaces

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,38 +4,41 @@
...
 - INTMDB-334	Determine best path forward for GCP PSC timeouts and implement [\#859](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/859)
-- INTMDB-364	Add support for serverless private endpoints (AWS) [\#314](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/314)
 - INTMDB-370	Docs not complete for cloud_backup_snapshot_restore_job	[\#870](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/870)
-- INTMDB-373	Add new notification parameters to the mongodbatlas_alert_config resource [\#883](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/883)	
-- INTMDB-377	Release 1.5 (both pre and then GA) [\#](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/)
-- INTMDB-378	Document for users how to get a pre-existing container id [\#883](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/883)
-- INTMDB-394	MS Teams alert support in terraform provider is missing	[\#](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/)
+- INTMDB-403  Update third_party_integration.markdown [\#851](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/851)
```
Comment (themantissa) [inline CHANGELOG.md:21]: Another space issue - not blocking

**Review by martinstibbe — APPROVED** [review]
LGTM

**Review by themantissa — APPROVED** [review]
LGTM - just a few nits that can be addressed later.

File: CHANGELOG.md (hunk 1 above)
Comment (Zuhairahmed) [reply CHANGELOG.md:10]: updated

File: CHANGELOG.md (hunk 2 above)
Comment (Zuhairahmed) [reply CHANGELOG.md:21]: updated

**Pull request merged**